tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -140,9 +143,23 @@ do
 	info "$(kubectl get pods --all-namespaces 2>&1)"
 	info "Executing ${K8S_TEST_ENTRY}"
 	# Output file will be prefixed with either "ok" or "not_ok" in case of
-	# success or fail, respectively.
+	# success or fail, respectively. If retry is enable and failed tests
```

Comment:
how about `Output file will be prefixed with "ok", "not_ok" or "flaky" based on the result` (I'd expect devels to know what flaky means but if you want to be explicit you can say `Output file will be prefixed with "ok", "not_ok" - pass/fail or "flaky" when it fails but passes retry.` (or something better, I'm not a native)